### DISCUSSION TITLE
How to export Rules to install on new computer ?

### DISCUSSION THREAD

**Discussion by shamantao** [objective-see/LuLu#587] [Q&A] [answered]
Hello,
I would like to clean my Mac and reinstall it. How can I export the rules i made in LULU to import it in new installation ?
Thanks

**Comment by mdjunior** [comment, marked as answer]
LuLu does not have a rules export functionality (for now), but the rules file follows a format that is compatible between versions. In other words, I believe that if you copy the rules file from the old computer to the new computer, it will work.

The path for rules file is: `/Library/Objective-See/LuLu/rules.plist`

**Just one point of attention:** you will need to disable LuLu and close it to then copy the file. Only after this can you start LuLu again to load the new file. It is important to be careful not to corrupt the data.

**Reply by shamantao** [reply]
This is perfect.
Thank you

**Reply by mdjunior** [reply]
If you are on a recent version of LuLu (above 2.6.0), it has a function in the menu to export and import rules. But keep in mind that it is a new feature.

**Reply by shamantao** [reply]
Oh thanks, Now I got it. My 2.5.1 version always said it was the last one. So I didn't update. 
I did now.

**Reply by Maghsad** [reply]
Same issue. The app auto-updater, as well as the official website, says 2.5.1 is the latest version. Is updating LuLu to version 2.6.2 safe by replacing the app files? i.e., is 2.6.2 compatible with 2.5.1 custom rules?

**Reply by shamantao** [reply]
Yes. I just did and the rules are the same and compatibles. Everything is fine

**Reply by Maghsad** [reply]
Thanks for letting me know.

**Comment by objective-see** [comment]
Happy to hear this is all working now. 
There have been some recent improvements to importing/exporting rules, but yes this should now all be working well! 

If you encounter any issues, please file a bug report and I'll work on getting it fixed ASAP!
